qml/DynamicDisplayStatisticsDialog.qml

```diff
@@ -39,6 +39,10 @@ Dialog {
     property var availableStatisticKinds: []
 
     Component.onCompleted: {
+        standardButton(Dialog.Apply).text = qsTrId("Add")
```

Comment:
Maybe better: `Apply`, `Apply & Close`, `Close` ?

Reply:
I cannot change that